src/main/webapp/app/overview/course-exams/course-exams.component.ts

```diff
@@ -60,4 +109,40 @@ export class CourseExamsComponent implements OnInit, OnDestroy {
     isVisible(exam: Exam): boolean {
         return exam.visibleDate ? dayjs(exam.visibleDate).isBefore(this.serverDateService.now()) : false;
     }
+
+    /**
+     * Returns if the given exam is a TestExam
+     * @param exam the exam to be checked
+     */
+    isTestExam(exam: Exam): boolean {
+        return exam.testExam ? exam.testExam : false;
+    }
+
+    /**
+     * Filters the studentExams for the examId and sorts them according to the studentExam.id in reverse order
+     * @param examId the examId for which the StudentExams should be retrieved
+     */
+    getStudentExamForExamIdOrderedByIdReverse(examId: number): StudentExam[] {
+        if (!this.studentExams) {
+            return [];
+        }
+        return this.studentExams
+            .filter(function (studentExam) {
+                return studentExam.exam?.id && studentExam.startedDate && studentExam.exam.id === examId && studentExam.startedDate;
+            })
+            .sort((se1, se2) => {
+                if (se1.id! > se2.id!) {
+                    return -1;
+                }
+                if (se1.id! < se2.id!) {
+                    return 1;
+                }
+                return 0;
+            });
```

Comment:
Seems like this is the same comparison function as somewhere above (already marked). Could reuse again

Reply:
It is not the exact method, as we're sorting by Id here and by startedDate above. But I shortened the sorting-method to `.sort((se1, se2) => se2.id! - se1.id!);`